core/zip/src/crc32.c

```diff
@@ -21,12 +21,18 @@
   DYNAMIC_CRC_TABLE and MAKECRCH can be #defined to write out crc32.h.
  */
 
-#if(defined __aarch64__ &&  defined __ARM_ACLE)
+#if(defined __aarch64__)
```

Comment:
If there is a commit from Cloudfare that has these changes, it would be nice to have a reference.
By adding a link I mean to just add in the commit message somewhere the original repo and commit hash in case it exists. If you made the changes, then add an explanation of why you needed the changes. I just want to understand what comes from cloudflare and what is adaptation for our needs, so that in the next update I can know what I will need to rebase, or how to resolve merge conflicts.

Reply:
Sure! This is PR: https://github.com/cloudflare/zlib/commit/f04f4ed63ae039f1d1bc5c0e3a57aa66e1e2cd6f (I made modifications on top to make it more flexible for various cases). Also, by the way, these changes will go also to cms-externals/zlib.

And soon as I know, I am still working with you and will stay around, so I could help you with rebase or merge.

Reply:
Hi Oksana, could you please add the references to the commit messages? I know you will be around for a while, but it's easy to add the refences with `git commit --amend` or with `git rebase -i` then changing `pick` to `reword` to just change the commit message. That can be helpful even if you do the rebase yourself later, as you may not remember all details. Thanks.

Reply:
I think you misunderstood what I did, so I will add reference only to code from cloudflare and cms-externals.

Reply:
Yes, that's what I mean, adding pointers to code that comes from elsewhere (even if you developed that code yourself) to the commits that introduce the upstream changes into ROOT.